src/main/resources/lang/default.lang

```diff
@@ -1953,6 +1953,19 @@ quit reasons:
 	kicked: kicked
 	timed_out: timed out
 
+# -- Transform Reasons --
+transform reasons:
+	cured: zombie cure, zombie curing, zombie cured, cured
+	drowned: entity drowned, drowned, zombie converting to drown, zombie drown
```

Comment:
The entity is called a Drowned, not a Drown
```suggestion
	drowned: entity drowned, drowned, zombie converting to drowned, zombie drown
```
The tenses here confuse me, you're using "entity drowned" (past tense) and "converting" (present tense)